From 44ae49c5a5b06bacd3f42256805b9758bc751361 Mon Sep 17 00:00:00 2001
From: Guanyu Chen <guanyuc@qti.qualcomm.com>
Date: Fri, 8 May 2026 17:35:38 +0800
Subject: [PATCH] serial: qcom-geni: Increase UART ports to 7

Increase the UART ports to 7 as use-case needs more than 5
UART ports.

Change-Id: I6d63f426155372a9b3f8f0b9f26531d02f69a166
Signed-off-by: Guanyu Chen <guanyuc@qti.qualcomm.com>
---
 drivers/tty/serial/qcom_geni_serial.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/tty/serial/qcom_geni_serial.c b/drivers/tty/serial/qcom_geni_serial.c
index 0723fc528e51..3e72425df100 100644
--- a/drivers/tty/serial/qcom_geni_serial.c
+++ b/drivers/tty/serial/qcom_geni_serial.c
@@ -96,7 +96,7 @@ void serial_trace_log(struct device *dev, const char *fmt, ...)
 #define STALE_TIMEOUT			16
 #define DEFAULT_BITS_PER_CHAR		10
 #define GENI_UART_CONS_PORTS		1
-#define GENI_UART_PORTS			5
+#define GENI_UART_PORTS			7
 #define DEF_FIFO_DEPTH_WORDS		16
 #define DEF_TX_WM			2
 #define DEF_FIFO_WIDTH_BITS		32
-- 
2.34.1

